packages/paste-core/components/pagination/src/PaginationArrow.tsx

```diff
@@ -0,0 +1,107 @@
+import * as React from 'react';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import {Text} from '@twilio-paste/text';
+import {ArrowBackIcon} from '@twilio-paste/icons/esm/ArrowBackIcon';
+import {ArrowForwardIcon} from '@twilio-paste/icons/esm/ArrowForwardIcon';
+import type {PaginationArrowProps} from './types';
+import {PaginationArrowPropTypes} from './proptypes';
+import {handleLinkValidation} from './utils';
+
+const PaginationArrow = React.forwardRef<HTMLButtonElement, PaginationArrowProps>(
+  (
+    {
+      as = 'button',
+      children,
+      direction = 'left',
+      href,
+      nextArrowLabel = 'Go to next page',
+      previousArrowLabel = 'Go to previous page',
```

Comment:
I could technically set `nextArrowLabel` **and** `previousArrowLabel` and I probably shouldn't be able to

Reply:
On further thought we might need to think a little harder on the API of this one.

Right now if I set 

```
<PaginationArrow nextArrowLabel="Go to next page">Next</PaginationArrow>
```

The accessible text is "Go to nextpageNext". So not only do we have duplication for the consumer but also for the customer.

The fact I can also do

```
<PaginationArrow nextArrowLabel="Go to next page" previusArrowLabel="Go to previous page">Next</PaginationArrow>
```

Feels weird.

First and foremost we need to think of the customer experience to assistive tech users. How can we tidy this API, and make clear button text for the customer?

Reply:
Included a fix for this in the PaginationArrow refactor.